pkg/analyze/selector/engine/relation_test.go

```diff
@@ -0,0 +1,59 @@
+package engine
+
+import (
+	"testing"
+)
+
+type RelationCheckTest struct {
+	name            string
+	firstParameter  string
+	secondParameter string
+	lambda          any
+	firstArgument   any
+	secondArgument  any
```

Comment:
Not important. firstParameter and firstArgument seem unrelated, but the former is the name of a variable that holds firstArgument, right?